db/organization.js

```diff
@@ -126,7 +134,8 @@ class OrganizationDbController {
       $set: {
         monthlyDonation: donationInMc !== 0,
         donationAmount: donationInMc,
-        globalDonation
+        ...(globalDonation && { globalDonation }),
+        ...(publicallyGive && { publicallyGive })
```

Comment:
I don't think this idea works as well when the property in question is a bool. Imagine calling this function like `setDonation({ orgId, amount, publicallyGive: false })`, intending to privatize giving... it would not act as expected. One solution is to change the left hand side of the conditional to `typeof publicallyGive !== 'undefined'`